code/datums/mutations/hulk.dm

```diff
@@ -28,7 +31,7 @@
 	if(source.a_intent != INTENT_HARM)
 		return
 	if(target.attack_hulk(owner))
-		if(world.time < 20)
+		if(world.time > (last_scream + scream_delay))
```

Comment:
You'll have to add a `last_scream = world.time` line afterward for this to work.